parm/config.orion.yaml

```diff
@@ -15,7 +15,7 @@ obsforge:
 
 aoddump:
   provider: VIIRSAOD
-  platforms: ['npp', 'n20', 'n21']
+  platforms: ['npp', 'j01', 'n21']
```

Comment:
This one is annoying - some NESDIS products use **n20** and others **j01**.
I am aware of it, but if you think it will help future developers, add a comment line to document this.